ui/jasmine_test/spec/test.spec.js

```diff
@@ -0,0 +1,20 @@
+import {AudioRecorder} from '../../src/AudioRecorder.js';
+
+describe("Testing the Audio Recorder methods", function() {
+
+    it("Start recording called", function(){
+        let audioRecorder = new AudioRecorder();
+        spyOn(audioRecorder, "startRecording");
+        audioRecorder.startRecording();
+        expect(audioRecorder.startRecording).toHaveBeenCalled();
+    });
+
+    it("Stop recording called", function(){
```

Comment:
ditto arrow function and ditto above comment to test the implementation of this function